apps/frontend/src/app/pages/person/user-profile/components/expertise/expertise.component.ts

```diff
@@ -0,0 +1,35 @@
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { NgFor } from '@angular/common';
+import { FormFieldComponent } from 'apps/frontend/src/app/shared/components/form-field/form-field.component';
```

Comment:
Instead of using absolut path use relative path:
![image](https://github.com/Venture-Miner/wagademy/assets/90660988/9e2a6fe4-192e-4646-9146-b20afd1534cb)